### DISCUSSION TITLE
Fehlermeldung

### DISCUSSION THREAD

**Discussion by trollingtom** [syssi/esphome-jk-bms#200] [Q&A]
Could not find the package with 'platformio/toolchain-esp32ulp @ ~1.22851.0' requirements for your system 'linux_aarch64'

was mache ich falsch?

Regards from Germany

Thomas

**Comment by syssi** [comment]
Das ESP-IDF-Framework lässt sich zurzeit nicht auf einem Raspberry Pi (aarch64 Architektur) bauen. Auf einem x86-Prozessor solltest du keine Probleme haben das Projekt zu kompilieren.

**Reply by trollingtom** [reply]
Ich habe nur den einen mit der aarch64 Architektur.....

**Reply by syssi** [reply]
ESPHome lässt sich auch Notfalls auf einer Windows-Maschine installieren. Einfacher ist eine Linux-Maschine.

**Reply by trollingtom** [reply]
Ist mir eben auch gerade eingefallen ich Dussel.

**Comment by fliegenderfish** [comment]
Blöde Frage, lässt sich das ganze eigentlich auch auf einem Raspberry Pi 4 mit Venus OS via Bluethooth nutzen? ganz ohne einem ESP Board usw.? Merci

**Reply by syssi** [reply]
Nein. Diese Implementierung nutzt ESPHome als "Unterbau". ESPHome ist mit wenigen ausgewählten Mikrocontrollern kompatibel und ist einfach gesprochen das Betriebssystem. Dein Raspberry Pi / Venus OS basiert auf Linux.

**Reply by fliegenderfish** [reply]
OK Danke für die Info, dann habe ich das mit dem RaspberryPi compilieren falsch verstanden....
Weißt du ein gleichwertiges tolles Projekt, das auch Linux basierend mit BLE funktioniert?
